test/testdata/interface_axelar_gateway_gateway.json

```diff
@@ -0,0 +1,82 @@
+{
+  "structs": {
+    "Gateway": {
+      "name": "Gateway",
+      "abilities": [
+        "key"
+      ],
+      "fields": [
+        {
+          "name": "id",
+          "type": "UID"
+        },
+        {
+          "name": "inner",
+          "type": "Versioned"
+        }
+      ]
+    },
+    "CreatorCap": {
+      "name": "CreatorCap",
+      "abilities": [
+        "store",
+        "key"
+      ],
+      "fields": [
+        {
+          "name": "id",
+          "type": "UID"
+        }
+      ]
+    }
+  },
+  "publicFunctions": {
+    "prepare_message": {
+      "name": "prepare_message",
+      "visibility": "public",
+      "params": {
+        "channel#0#0": "&Channel",
+        "destination_chain#0#0": "String",
+        "destination_address#0#0": "String",
+        "payload#0#0": "vector<u8>"
```

Comment:
just wondering: why do params have a `#0#0` postfix here? could we remove it?

Reply:
no idea, i'd like to know in case it changes for some reason so i don't want to remove it.